### DISCUSSION TITLE
Multiple Store use case

### DISCUSSION THREAD

**Discussion by christopherperry** [reduxkotlin/redux-kotlin#142] [General]
Hello, I work at Uber and we use the RIBs architecture. Parts of our application come and go and the application codebase is extremely large and complex. While reading your docs I noticed several times that it is recommended to use a single Store to hold the entire application state. What would be the issue if we decided to use one Store per RIB in the application?

**Comment by trOnk12** [comment]
hey @christopherperry any feedback on this ? Did you decided to implement Redux ? Any feedback would be awesome :) Do you have any idea how to connect it with multiple modules ? Do you consider to have one module which all the features depend on ?
